src/modules/login/login.js

```diff
@@ -270,7 +281,11 @@ export function sanitizeLoginStash (stashTree: LoginStash, appId: string) {
     stash => stash.appId === appId,
     stash => stash,
     (stash, children) => {
-      const login = filterObject(stash, ['username', 'appId', 'loginId'])
+      const login: LoginStash = filterObject(stash, [
+        'username',
+        'appId',
+        'loginId'
+      ])
       login.children = children
```

Comment:
Does this need an elvis statement like above?

Reply:
No. In this case, Flow is happy with things. `LoginStash.children` is allowed to be null.